legal-api/src/legal_api/resources/v2/internal_services.py

```diff
@@ -89,3 +96,26 @@ def create_registrars_notation_filing(business: Business, user: User, old_bn: st
     filing.save()
 
     return ListFilingResource.complete_filing(business, filing, False, None)
+
+
+@bp.route('/statistics', methods=['GET'])
+@cross_origin(origin='*')
+@jwt.has_one_of_roles([UserRoles.staff])
+def get_statistics():
+    """Return a JSON object with statistic information."""
+    view_type = request.args.get('view', None)
+
+    if not view_type:
+        return ({'message': 'Invalid query parameter in request.'}, HTTPStatus.BAD_REQUEST)
+
+    data = {}
+    if view_type == ViewTypes.DISSOLUTION:
+        count = InvoluntaryDissolutionService.get_businesses_eligible_count()
+        data = {
+            **data,
+            'eligibleCount': count
+        }
+
+    return jsonify({
+        'data': data
+    }), HTTPStatus.OK
```

Comment:
Have been thinking about this more.  I think I was trying to make this too generic.

Let's do away with the concept of "views" for now and create an endpoint under `/admin/dissolutions/statistics`.

And in the future if we need to retrieve dissolution batch info, we can add additional endpoints under `admin/dissolutions`.  e.g. `/admin/dissolution/batches`

Also, this addresses @vysakh-menon-aot 's suggestion to move the endpoint to a more appropriate location too.